projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -196,7 +197,9 @@ export class TuiInputPhoneComponent
         }
     }
 
-    onBackspace(event: Event & {target: HTMLInputElement}) {
+    onBackspace(keyboardEvent: KeyboardEvent) {
+        const event = keyboardEvent as unknown as Event & {target: HTMLInputElement};
```

Comment:
I think it's better to typecast target here.

Reply:
fixed